test/api/visualization/annotations_controller_test.rb

```diff
@@ -0,0 +1,98 @@
+require 'api_test_helper'
+require 'test_instrumentor'
+
+class AnnotationsControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+  include Requests::JsonHelpers
+  include Requests::HttpHelpers
+  include Minitest::Hooks
+    include TestInstrumentor
+
+  before(:all) do
```

Comment:
VERY cool - no more `setup` and `teardown`!  This should be a major time savings.